virttest/utils_libvirt/libvirt_disk.py

```diff
@@ -66,8 +67,6 @@ def get_non_root_disk_names(session):
     s, o = utils_misc.cmd_status_output(
         cmd, ignore_status=False, shell=True, verbose=True, session=session
     )
-    if s:
```

Comment:
I think you shouldn't remove these lines but do
```
if s and not ignore_status:
    raise ....
```
to maintain existing behavior.

Reply:
+1

Reply:
ah... yes,  I just realize ignore_status does not take effect in  utils_misc.cmd_status_output() when session is not None.

Updated.